package.json

```diff
@@ -17,7 +17,7 @@
     "classnames": "2.2.5",
     "electron-updater": "2.21.4",
     "ethereum-blockies-base64": "1.0.1",
-    "ethereumjs-abi": "0.6.5",
+    "ethereumjs-abi": "git://github.com/ethereumjs/ethereumjs-abi.git#09c3c48fd3bed143df7fa8f36f6f164205e23796",
```

Comment:
We should bug them about releasing a new version if there's something that's needed but isn't out yet. Pegging to a commit has caused us build woes in the past.

Reply:
I asked kindly for drafting new release.  :)

Reply:
Seems like at this moment it won't be possible according to repository maintainer, he doesn't have contact to main maintainer.

https://github.com/ethereumjs/ethereumjs-abi/pull/60#issuecomment-379768953